From: Juergen Beisert <jbe@pengutronix.de>
Date: Wed, 28 Jan 2015 19:02:20 +0100
Subject: [PATCH] Just cosmetic

Keep this display as number 6, to be in sync with the older kernels

Signed-off-by: Juergen Beisert <jbe@pengutronix.de>
---
 arch/arm/mach-s3c24xx/mach-mini2440.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/arm/mach-s3c24xx/mach-mini2440.c b/arch/arm/mach-s3c24xx/mach-mini2440.c
index 5cc40ec1d254..92e287f04ffb 100644
--- a/arch/arm/mach-s3c24xx/mach-mini2440.c
+++ b/arch/arm/mach-s3c24xx/mach-mini2440.c
@@ -169,8 +169,7 @@ static struct s3c2410fb_display mini2440_lcd_cfg[] __initdata = {
 		.lcdcon5	= (S3C2410_LCDCON5_FRM565 |
 				   S3C2410_LCDCON5_HWSWP),
 	},
-	/* mini2440 + 3.5" TFT (LCD-W35i, LQ035Q1DG06 type) + touchscreen*/
-	[3] = {
+	[6] = { /* LCD-W35i 3.5" display (LQ035Q1DG06)*/
 		_LCD_DECLARE(
 			/* clock */
 			7,
